src/base/bittorrent/torrent.h

```diff
@@ -84,6 +86,18 @@ namespace BitTorrent
         Error
     };
 
+    class CustomNaturalLessThan
```

Comment:
It's too generic a name to be public in BitTorrent namespace. I would rename it to something like `TagLessThan`.